themes/digitalservices/layouts/partials/header.html

```diff
@@ -35,3 +35,10 @@
         class="jobs"
     {{ end }}
 >
+    <!-- Screendoor Setup -->
+    <script>window.jQuery || document.write('<script src="//code.jquery.com/jquery-2.2.3.min.js"><\/script>')</script>
+    <link href="//d3q1ytufopwvkq.cloudfront.net/1/formrenderer.css" rel="stylesheet" /> 
+    <script src="//d3q1ytufopwvkq.cloudfront.net/1/formrenderer.js"></script> 
+    <link rel="stylesheet" href="/css/styles.css">
+    <!-- End Screendoor Setup -->
+</body>
```

Comment:
You won't want to add `</body>` here as this partial expects there to be more content after. I'm actually a little surprised to see the beginning `<body>` tag in this file. I would have expected it in the layout. If you don't feel like refactoring that bit, the simplest thing to do would be to move it to the `footer.html` partial. Also, I believe you want to leave the CSS here, just move the JS down.

Reply:
On a different note ` <link rel="stylesheet" href="/css/styles.css">` is already in this file so you don't need to add it again 😄

Reply:
in @akegan - that wasn't her.  it was me! 😢 .  when i tried to fix the asset references.  see https://github.com/SFDigitalServices/sfdigitalservices.github.io/commit/1fcc9643415d189a135995ddb4aadb70537d5c7b#diff-abdd46c1512134cf4cd13003be774c2f

I removed the filter function and forgot to remove the dup

Reply:
It's all good, I know it wasn't part of this change set, but just noticed the comment too.